### PR TITLE
bug/1898_no_multiservice_and_servicepath

### PR DESCRIPTION
**Pull request by kzangeli**
Fixes issue #1898

When the broker is started without -multiservice, the service path is no longer used as a filter for subscriptions. The broker simply **ignores** the service path.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -11,3 +11,4 @@
 - Fix: using "/#" as default service path instead of "/" in subscriptiobns (#1961)
 - Fix: POST /v2/subscriptions, attributes in notification are optional (#1952)
 - Add: Correlator as HTTP header and in log file (Issue #1916)
+- Fix: no longer considering service-path for subscriptions when broker is started without '-multitenant' CLI option (Issue #1898)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: no longer considering service-path... -> no longer considering service

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Fixed in f4491b9

**Comment by fgalan** [comment]
LGTM (passing the ball to @crbrox for final LGTM + merge + delete branch)

**Comment by crbrox** [comment]
LGTM

**Pull request merged**